Client/game_sa/CModelInfoSA.h

```diff
@@ -466,7 +466,7 @@ class CModelInfoSA : public CModelInfo
     // Vehicle towing functions
     bool IsTowableBy(CModelInfo* towingModel) override;
 
-    bool IsDynamic() { return m_pInterface ? m_pInterface->usDynamicIndex != 0xffff : false; };
+    bool IsDynamic() { return m_pInterface ? m_pInterface->usDynamicIndex != MODEL_PROPERTIES_GROUP_STATIC : false; };
```

Comment:
Maybe const?
I know you arent introducing a new function but if you're at refactoring then you can as well mark it as const